src/msix/PAL/Crypto/Win32/Crypto.cpp

```diff
@@ -14,21 +14,60 @@
 
 #include <memory>
 #include <vector>
-
-struct unique_hash_handle_deleter {
-    void operator()(BCRYPT_HASH_HANDLE h) const {
-        BCryptDestroyHash(h);
+#include <deque>

```

Comment:
I don't agree with the change here; either our usage of unique_ptr is flawed and should be fixed, or this is unnecessary.  Adding this additional overhead isn't the right answer.